Sources/MSLUIKit/UIViewController+Transition.swift

```diff
@@ -0,0 +1,263 @@
+import UIKit
+
+/// When using Coordinators, you'll find that you may want to swap out one view controller for another. Instead of
+/// this swap having no animation, this extension helps restore animation with the changing of views.
+///
+/// Example:
+///
+/// ```
+/// self.rootViewController.present(
+///    to: self.caseListViewController,
+///    withOptions: .init(style: .swipe, direction: .right)
+/// )
+/// ```
+public extension UIViewController {
```

Comment:
This file starts out with a nice comment but then there just aren't any after that... could we add some more?